packages/components/pages/f-account-info/src/components/_tests/AccountInfo.test.js

```diff
@@ -251,6 +290,46 @@ describe('AccountInfo', () => {
                     // Assert
                     expect(wrapper.vm.hasFormUpdate).toEqual(true);
                 });
+
+                describe('form field is part of the address', () => {
+                    it.each([
+                        ['line1'],
+                        ['line2'],
+                        ['line3'],
+                        ['postcode'],
+                        ['locality']
+                    ])('should set `hasAddressBeenUpdated` to `true` when `field` is %s', async field => {
+                        // Arrange
+                        wrapper = await mountAccountInfo();
+
+                        const element = wrapper.find(`[data-test-id="account-info-consumer-${field}"]`);
+
+                        // Act
+                        element.vm.$emit('input', 'Nice Road');
+
+                        // Assert
+                        expect(wrapper.vm.hasAddressBeenUpdated).toEqual(true);
+                    });
+                });
+
+                describe('form field is not part of the address', () => {
+                    it.each([
+                        ['firstName'],
+                        ['lastName'],
+                        ['phoneNumber']
+                    ])('should not set `hasAddressBeenUpdated` to `true` when `field` is %s', async field => {
+                        // Arrange
+                        wrapper = await mountAccountInfo();
+
+                        const element = wrapper.find(`[data-test-id="account-info-consumer-${field}"]`);
+
+                        // Act
+                        element.vm.$emit('input', 'Harry Potter');
```

Comment:
Might be better to use a valid phone number?
```suggestion
                    it.each([
                        ['firstName', 'Harry'],
                        ['lastName', 'Potter'],
                        ['phoneNumber', '07890123456']
                    ])('should not set `hasAddressBeenUpdated` to `true` when `field` is %s', async (field, inputValue) => {
                        // Arrange
                        wrapper = await mountAccountInfo();

                        const element = wrapper.find(`[data-test-id="account-info-consumer-${field}"]`);

                        // Act
                        element.vm.$emit('input', inputValue);
```